presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1538,8 +1540,8 @@ public void testMaterializedViewForJoinWithMultiplePartitions()
         }
     }
 
-    @Test(enabled = false)
-    public void testMaterialziedViewFullOuterJoin()
+    @Test
```

Comment:
accidental change?

Reply:
I believe this test works because it expects an exception message and doesn't compare the view vs. base tables. Also, I changed the function name because of a typo in the original code; if it affects anything else, I can change it back.

Reply:
This sounds good to me. thanks for making this change.